<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="fileScanButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string>File Scanner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stegButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Steganography</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QPushButton" name="imageButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="messageTextEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>281</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="encodeButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>340</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Encode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decodeButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>450</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Decode</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>281</width>
        <height>256</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>281</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="decodeLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>420</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="fileButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hashButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Hash</string>
      </property>
     </widget>
     <widget class="QPushButton" name="compareButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>290</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Compare File</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCybersec_Toolkit">
    <property name="title">
     <string>Cybersec Toolkit</string>
    </property>
   </widget>
   <addaction name="menuCybersec_Toolkit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
